### DISCUSSION TITLE
Práctica JS

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#381] [Q&A]
Buenas noches, ¿la práctica de JS, es hacer la pokedex, pero ponerle estilo y más cosas como que muestre las características del Pokemon?. 

Por cierto no se todo de Pokemon, pero si lo vi en mi infancia y me gustaba. Mi pokemon favorito es el Eevee

**Comment by doguedogue** [comment]
Pues es lo que comentó el MC sobre el pokedex, pero no ha dado las especificaciones oficiales, hacer la pokedex va a estar genial, es el aparato que miraban para revisar los pokemones, no? adelante compañera, entre más practicas mejor🚀

**Reply by GloriaCoralCerecedo** [reply]
Oh bueno, si así es para saber que tipo de pokemon es, gracias!

**Reply by doguedogue** [reply]
A mí me gusta Psycho duck jajajaj

**Comment by hackit92** [comment]
Si justo eso, está siendo una práctica muy divertida y un tanto compleja respecto al diseño
